### ISSUE TITLE
Razer Viper 8kHz Polling Rate

### ISSUE THREAD

**Issue by StefanW0725** [1kc/razer-macos#490]
Hi! I have a Razer Viper 8kHz mice. I tried to change the polling rate of it using this razer-macOS today, but it doesn't seem to work. The initial polling rate should be 8kHz or something but 0 Hz is shown in the software. Also, when I tried to change it to 1000 Hz for example, it didn't actually change and memorize it. Could you try to come up with a solution of it, please? Thank you so much!

**Comment by WrathofthePast** [comment]
@StefanW0725, yes. I'm working on it. openrazer just got the 8kHz working. Are you interested in testing it?

**Comment by sfroment** [comment]
hey, I've already done a bit here https://github.com/1kc/librazermacos/issues/6

**Comment by WrathofthePast** [comment]
@sfroment, I saw. I have a counter proposal though. I think `razerchromacommon.h` & `razerchromacommon.c` should be made into a dependency instead as they're shared between razer-macos & openrazer.
